### ISSUE TITLE
"npm run minify" not working

### ISSUE THREAD

**Issue by yairEO** [punkave/sanitize-html#148]
````
0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'minify' ]
2 info using npm@3.10.10
3 info using node@v6.9.4
4 verbose run-script [ 'preminify', 'minify', 'postminify' ]
5 info lifecycle sanitize-html@1.14.1~preminify: sanitize-html@1.14.1
6 silly lifecycle sanitize-html@1.14.1~preminify: no script for preminify, continuing
7 info lifecycle sanitize-html@1.14.1~minify: sanitize-html@1.14.1
8 verbose lifecycle sanitize-html@1.14.1~minify: unsafe-perm in lifecycle true
9 verbose lifecycle sanitize-html@1.14.1~minify: PATH: C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin;C:\www\node_modules\sanitize-html\node_modules\.bin;C:\www\node_modules\.bin;C:\Ruby22-x64\bin;C:\ProgramData\Oracle\Java\javapath;C:\windows\system32;C:\windows;C:\windows\System32\Wbem;C:\windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Common Files\EMC\;C:\Program Files\nodejs\;C:\Program Files\Git\cmd;C:\Program Files\TortoiseGit\bin;C:\apache-maven-3.2.5\bin;C:\Program Files\Git\bin;C:\Users\evenay\AppData\Roaming\npm
10 verbose lifecycle sanitize-html@1.14.1~minify: CWD: C:\www\node_modules\sanitize-html
11 silly lifecycle sanitize-html@1.14.1~minify: Args: [ '/d /s /c',
11 silly lifecycle   'npm run build && uglifyjs dist/sanitize-html.js > dist/sanitize-html.min.js' ]
12 silly lifecycle sanitize-html@1.14.1~minify: Returned: code: 1  signal: null
13 info lifecycle sanitize-html@1.14.1~minify: Failed to exec minify script
14 verbose stack Error: sanitize-html@1.14.1 minify: `npm run build && uglifyjs dist/sanitize-html.js > dist/sanitize-html.min.js`
14 verbose stack Exit status 1
14 verbose stack     at EventEmitter.<anonymous> (C:\Program Files\nodejs\node_modules\npm\lib\utils\lifecycle.js:255:16)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at EventEmitter.emit (events.js:191:7)
14 verbose stack     at ChildProcess.<anonymous> (C:\Program Files\nodejs\node_modules\npm\lib\utils\spawn.js:40:14)
14 verbose stack     at emitTwo (events.js:106:13)
14 verbose stack     at ChildProcess.emit (events.js:191:7)
14 verbose stack     at maybeClose (internal/child_process.js:877:16)
14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
15 verbose pkgid sanitize-html@1.14.1
16 verbose cwd C:\www\node_modules\sanitize-html
17 error Windows_NT 6.1.7601
18 error argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "minify"
19 error node v6.9.4
20 error npm  v3.10.10
21 error code ELIFECYCLE
22 error sanitize-html@1.14.1 minify: `npm run build && uglifyjs dist/sanitize-html.js > dist/sanitize-html.min.js`
22 error Exit status 1
23 error Failed at the sanitize-html@1.14.1 minify script 'npm run build && uglifyjs dist/sanitize-html.js > dist/sanitize-html.min.js'.
23 error Make sure you have the latest version of node.js and npm installed.
23 error If you do, this is most likely a problem with the sanitize-html package,
23 error not with npm itself.
23 error Tell the author that this fails on your system:
23 error     npm run build && uglifyjs dist/sanitize-html.js > dist/sanitize-html.min.js
23 error You can get information on how to open an issue for this project with:
23 error     npm bugs sanitize-html
23 error Or if that isn't available, you can get their info via:
23 error     npm owner ls sanitize-html
23 error There is likely additional logging output above.
24 verbose exit [ 1, true ]
````

**Comment by Harvinator** [comment]
I had to change the scripts section of package.json to this (for windows)

```
  "scripts": {
    "build": "browserify index.js > dist/sanitize-html.js --standalone 'sanitizeHtml'",
    "minify": "npm run build && uglifyjs dist/sanitize-html.js > dist/sanitize-html.min.js",
    "test": "mocha test/test.js",
    "prebuild": "npm run test && rmdir dist /s /q && mkdir dist"
  },
```

Also i had to install the following packages

`npm install -g minifier`
`npm install -g mocha`
`npm install -g browserify`
`npm install -g uglify-js`

**Comment by boutell** [comment]
Resolved in 1.16.3.
